### PR TITLE
Split core.js. Add schemaloader.js module.

### PR DESCRIPTION
**Pull request by tohosaku**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks backward-compatibility?    | ✔️ some private method moved from core.js
| Tests pass?   | ✔️
| Fixed issues  | -
| Updated README/docs?   | ❌
| Added CHANGELOG entry?   | ❌

core.js contains codes that configure the entire editor and codes that loads the schema to editor.
The code that loads the schema is less dependent on other modules, so separate it into another module.

### CODE REVIEW

File: tests/unit/schemaloader.spec.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
 import { SchemaLoader } from '../../src/schemaloader'
+import * as sinon from 'sinon'
 
 describe('SchemaLoader', () => {
   let loader
   let fetchUrl
   let fileBase
 
   beforeEach(() => {
-    loader = new SchemaLoader()
```
Comment (schmunk42) [inline tests/unit/schemaloader.spec.js]: Is this a BC breaking change of the reworked SchemaLoder code?
Or should this be two tests, what is the difference when `ajax: true` - isn't it always AJAX?

Comment (tohosaku) [reply tests/unit/schemaloader.spec.js]: Divided the test code into cases where { ajax : true }  is required and cases where not.

**Comment by schmunk42** [comment]
@pmk65 @gbisurgi @amk0 Please have a second look.

**Comment by schmunk42** [comment]
Thank you!

**Pull request merged**